charts/cf-runtime/.ci/values-ci.yaml

```diff
@@ -23,15 +29,15 @@ runtime:
     limits:
         cpu: 1000m
         memory: 1024Mi
-    podAnnotations:
-      karpenter.sh/do-not-evict: 'true'
-    nodeSelector:
-      node-type: dind
-    tolerations:
-    - effect: NoSchedule
-      key: codefresh.io
-      operator: Equal
-      value: dinds
+    # podAnnotations:
+    #   karpenter.sh/do-not-evict: 'true'
+    # nodeSelector:
+    #   node-type: dind
+    # tolerations:
+    # - effect: NoSchedule
+    #   key: codefresh.io
+    #   operator: Equal
+    #   value: dinds
```

Comment:
Why not just remove this commented out code? It'll be in the git history either way